pythainlp/transliterate/thai2rom_onnx.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+"""
+Romanization of Thai words based on machine-learnt engine in ONNX runtime ("thai2rom")
+"""
+from pythainlp.corpus import get_corpus_path
+import numpy as np
+import json
+
+from onnxruntime import InferenceSession
+
+_MODEL_ENCODER_NAME = "thai2rom_encoder_onnx"
```

Comment:
My Initial attempt to convert the whole model to ONNX failed (convert success but model prediction result not matched with original model). So I seperately convert encoder and decoder.